app/api/development/route.ts

```
@@ -0,0 +1,44 @@
+  if (!isDevelopment) {
+    return new NextResponse('Not available in production', { status: 403 })
+  }
+
+  try {
+    const body = await request.json()
+    
+    // Handle test actions
+    switch (body.action) {
+      case 'test_calendar_connection':
+        // Test calendar connection using the dedicated endpoint
+        const res = await fetch('http://localhost:3000/api/development/google-calendar')
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)